pkgs/applications/misc/postrunner/default.nix

```diff
@@ -0,0 +1,21 @@
+# To run it call: nix-build -A postrunner.tests
+{ lib, bundlerApp, bundlerUpdateScript, callPackage }:
+
+bundlerApp {
+  pname = "postrunner";
+  gemdir = ./.;
+  exes = [ "postrunner" ];
+  passthru.updateScript = bundlerUpdateScript "postrunner";
+  passthru.tests = {
+    simple-execution = callPackage ./tests.nix { };
+  };
+
+
+  meta = with lib; {
+    description = "PostRunner is an application to manage FIT files produced by Garmin products";
+    homepage    = "https://github.com/scrapper/postrunner.git";
+    license     = licenses.gpl2Plus;
```

Comment:
https://github.com/scrapper/postrunner#license looks like `gpl2Only` to me?

Reply:
My understanding was, that with the GPL v2 you are entitled to use it also with a later version of the GPL. But I am not a license specialist.

But on github pkgs/tools/text/unoconv/default.nix which has licenses.gpl2 points to same license GPL v20 as the gpl2only marked in pkgs/tools/typesetting/htmldoc/default.nix.

And in https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix we have
``` 
 gpl2Only = {
    spdxId = "GPL-2.0-only";
    fullName = "GNU General Public License v2.0 only";
  };
```
I do not find any mentioning of GPL-2.0-only in the postrunner code
.
So I am confused. Do you now how can get a definitive answer?

Reply:
> My understanding was, that with the GPL v2 you are entitled to use it also with a later version of the GPL. But I am not a license specialist.

If https://github.com/scrapper/postrunner/blob/master/lib/postrunner.rb#L8-L10 would mention or later. So gpl2Only is correct.

Reply:
> Do you now how can get a definitive answer?

I think the license text is pretty clear: https://github.com/scrapper/postrunner/blob/master/COPYING#L242

```
Each version is given a distinguishing version number.  If the Program specifies a version number of this License which applies to it and "any later version", you have the option of following the terms and conditions either of that version or of any later version published by the Free Software Foundation.  If the Program does not specify a version number of this License, you may choose any version ever published by the Free Software Foundation.
```

So AFAICS that confirms that, since they do specify 'version 2' but don't specify 'or any later version', we should interpret that as `gpl2Only`.